gulp-tasks/publish-glitch.js

```diff
@@ -0,0 +1,67 @@
+/*
+  Copyright 2018 Google LLC
+
+  Use of this source code is governed by an MIT-style
+  license that can be found in the LICENSE file or at
+  https://opensource.org/licenses/MIT.
+*/
+
+const gulp = require('gulp');
+const childProcess = require('child_process');
+const fse = require('fs-extra');
+const tempy = require('tempy');
+const util = require('util');
+const del = require('del');
+
+const exec = util.promisify(childProcess.exec);
+
+const glitchProjects = [
+  'workbox-core',
+  // 'workbox-sw',
+  // 'workbox-precaching',
+  // 'workbox-strategies',
+  // 'workbox-background-sync-demo',
+  // 'workbox-routing',
+  // 'workbox-expiration',
+  // 'workbox-cacheable-response',
+  // 'workbox-google-analytics',
+  // 'workbox-streams',
+  // 'workbox-range-requests',
+  // 'workbox-broadcast-update-demo',
+  // 'workbox-window',
+  // 'workbox-navigation-preload'
+];
+
+
+gulp.task('publish-glitch', async () => {
+  if (!process.env.GLITCH_PERSONAL_TOKEN) {
```

Comment:
@jeffposnick I went for the scorched earth strategy and just rolled back to clean up all problematic history, and recommited everything at once. Instead of using fs-extra, I thought it'd make sense to follow the convention used in publish-github.js and use node environment vars. I noticed you don't have dotenv: do you just pass the env vars with the command?

Reply:
Okay, that's fine.

I've got some environment vars set in my `~/.bashrc` config.

Reply:
Sounds good. I'll push demo server changes later this week (since I'll be adding or editing server.js code for each demo), make sure this task works when deploying to every demo, and then we can merge